example/main.dart

```diff
@@ -1,17 +1,22 @@
-import 'package:http/http.dart' as http;
+import 'dart:io';
+
+import 'package:http/io_client.dart';
 import 'package:webfeed/webfeed.dart';
 
 void main() async {
-  var client = http.Client();
+  final client = IOClient(HttpClient()
+    ..badCertificateCallback =
+        ((X509Certificate cert, String host, int port) => true));
```

Comment:
This should be removed since it would be unsecure.

Reply:
reverted changes here